src/EditorFeatures/Test/MetadataAsSource/MetadataAsSourceTests.cs

```diff
@@ -2198,8 +2271,7 @@ void M()
 
 using System.Runtime.CompilerServices;
 
-public class [|TestType|]<[NullableAttribute(1)]
-T> where T : notnull
+public class [|TestType|]<[NullableAttribute(1)] T> where T : notnull
```

Comment:
Do we have a test of this behavior that doesn't involve NullableAttribute? The fact it's currently dumping this in metadata as source is something we plan on fixing, and if updating this test removes the coverage for the bug we'd want to test this some other way.

Reply:
Yes. I added a test that uses a custom attribute specifically for this purpose.